src/com/scottyplunkett/server/HTTPResponse.java

```diff
@@ -20,9 +19,8 @@ public static void setBody(String htmlString) {
 
     static String build(String requested) throws IOException {
         String route = Parser.findRequestedRoute(requested);
-        Path path = Paths.get(route);
-        String responseHeaders = buildHeaders("200", "OK",
-                                              "text/html");
+        int encoded = HTTPResponseCode.encode(route);
```

Comment:
I like what's going on here, and just after it - but, is `encoded` used anywhere else?  If not, then it might be an internal concern of `HTTPResponseCode`.  What would this code look like if you were creating an `HTTPResponseCode` object instance, and then just using it throughout a `HTTPResponse` instance?